x/assets/keeper/operator_asset.go

```diff
@@ -116,6 +116,11 @@ func (k Keeper) IteratorAssetsForOperator(ctx sdk.Context, operator string, asse
 		if err != nil {
 			return err
 		}
+		if isUpdate {
```

Comment:
It's better to check isUpdated firstly before open the storage

Reply:
No, it's a flag to indicate if the updated state should be stored

Reply:
But you do nothing while this flag is false instead only opening the storage blankly

Reply:
I can't understand your meaning. The storage is only opened once to get the state from the iterator. Then the retrieved state will be handled by the opFunc. The flag is used to indicate whether the state should be restored, so the storage should be opened whether the flag is true or false.